src/scripts/Tooltip.jsx

```diff
@@ -218,18 +219,21 @@ export default class JoyrideTooltip extends React.Component {
       header: {},
       hole: {},
       tooltip: {
-        position: cssPosition === 'fixed' ? 'fixed' : 'absolute',
+        position: isFixed ? 'fixed' : 'absolute',
         top: Math.round(yPos),
         left: Math.round(xPos)
       }
     };
 
+    const targetTop = isFixed ? opts.rect.top : (opts.rect.top - document.body.getBoundingClientRect().top);
+    const targetLeft = isFixed ? opts.rect.left : (opts.rect.left - document.body.getBoundingClientRect().left);
```

Comment:
+1 for caring about the left side. Sadly, the rest of his code does not account for any right/left scroll/body offset, but this is a start!